package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@theconversation/ui",
-  "version": "1.8.2",
+  "version": "1.9.0",
```

Comment:
My preference here is that we don't change the version in our feature PRs. We should have a separate PR which bumps the version and updates the changelog. The reason being is that we could potentially have conflicts if multiple people are building different features and they are bumping the version. What do you think?

Reply:
Agreed. Will remove!